build/AzurePipelinesTemplates/WindowsAppSDK-RunTests-Steps.yml

```diff
@@ -0,0 +1,115 @@
+parameters:
+  buildPlatform: ''
+  buildConfiguration: ''
+  ImageName: ''
+  TaefSelect: '*'
+  BinaryCompatSwitch: ''
+  testLocale: ''
+steps:
+  # Download the BuildOutput from the Build stage(s).
```

Comment:
Do we want to run TestAll.ps1 -AzureBuildStep DisplayInfo here?

Reply:
Do we need DisplayInfo?

Reply:
Or can I pull this out of TestAll and put it in RunTests-Steps.yml instead?

Reply:
The DisplayInfo code? 
I'll leave it up to you.

But I am noticing that we don't really have a need for "AzureBuildStep" in TestAll.ps1.
We introduced this parameter in the aggregator's code because we needed to run Azure tasks in between the code in  BuildAll or TestAll. 

Also one difference between the testall code in the aggregator and testall code in Foundation is that the test code gets built in Foundation's buildall code whereas the testcode in the aggregator get's built in the aggregator's testall code.

So maybe we should move building the foundation's test code from buildall to testall.

Reply:
"But I am noticing that we don't really have a need for "AzureBuildStep" in TestAll.ps1." +1

This is because there as two dev/test slns correct? I think we can move it out and let TestAll be pure testing for now.

Reply:
Ah yes. Don't need to be in this PR for sure. It will be for future refactors.

Reply:
Talk to @EricJohnson327

I'm fond of "build=build and test=test". If we need a "do everything" command then it should be something else

```
magic.ps1 -build ... ==> & build.cmd ...
magic.ps1 -test ... ==> & test.cmd ...
etc
```

Reply:
>move building the foundation's test code from buildall to testall

I'm fond of "build=build and test=test". If we need a "do everything" command then it should be something else

```
magic.ps1 -build ... ==> & build.cmd ...
magic.ps1 -test ... ==> & test.cmd ...
etc
```